help/en/html/doc/Technical/githubdesktopinto.shtml

```diff
@@ -0,0 +1,320 @@
+
+<br/><a href="images/GHD_Image_8_NewBranchName.png"><img src="images/GHD_Image_8_NewBranchName.png" height="50%" width="50%"></a><br/>
+
+It's not recommended to use international characters or spaces in the branch name. It
+may cause problems for the maintainers later.
+<P>
+It's possible to create the branch from another branch than the master branch. For
+example, JMRI may use the branch "development" as the branch to base developent on.
+If this is the case, you select the development branch, then merge the branch
+"upstream/development" into the branch "development" and then create the branch from
+development. Note that if you do this, GitHub desktop will warn you and ask if you
+really want to start from the development branch or start from the master branch.
+Select the "development" branch in this case.
+
+<br/><a href="images/GHD_Image_9_NewBranch_Development.png"><img src="images/GHD_Image_9_NewBranch_Development.png" height="50%" width="50%"></a><br/>
+
+When a new branch is created, it only exists on your local computer. In order to push
+it to the server, we click on the button "Publish branch".
+
+<br/><a href="images/GHD_Image_9_NewBranch_Development.png"><img src="images/GHD_Image_9_NewBranch_Development.png" height="50%" width="50%"></a><br/>
+
+<h3>Do a commit</h3>
+
+Using your favourite development tool, for example Eclipse or NetBeans, do the changes
+you want. For the purpose of this text, we create a simple class HelloWord.java.
+
+<br/><a href="images/GHD_Image_11_NewClass.png"><img src="images/GHD_Image_11_NewClass.png" height="50%" width="50%"></a><br/>
+
+GitHub Desktop will now show the changes we have done, in this case added a new class.
+We add a description for the commit that describes what this commit does and then click
+on the button "Commit to ...". The branch we are commiting to is "hello_world" so the
+button shows "Commit to hello_world".
+
+<br/><a href="images/GHD_Image_12_CreateCommit.png"><img src="images/GHD_Image_12_CreateCommit.png" height="50%" width="50%"></a><br/>
+
+We then click the button "Push origin" to upload our commit to the GitHub server. Note
+that this sends the commit to my repository, not the JMRI repository.
+
+<h3>Create a Pull Request, PR</h3>
+
+When we have done the commits we want, it's time to do a Pull Request, PR. It tells the
+maintainers of JMRI that you want this code to be included into JMRI.
+<P>
+In order to do that, we click on the button "Create Pull Request".
+
+<br/><a href="images/GHD_Image_13_CreatePullRequest.png"><img src="images/GHD_Image_13_CreatePullRequest.png" height="50%" width="50%"></a><br/>
+
+This will open a web browser. If you are not logged in to GitHub, you need to log in.
+When you have logged in, you will see the web page for creation of a Pull Request.
+
+<br/><a href="images/GHD_Image_13_CreatePullRequest.png"><img src="images/GHD_Image_13_CreatePullRequest.png" height="50%" width="50%"></a><br/>
```

Comment:
This should be `GHD_Image_14_CreatePullRequest.png` instead.

Reply:
`Image_14_CreatePullRequest.png` should be renamed to `GHD_Image_14_CreatePullRequest.png`

Reply:
I don't have a file by that name. Could it be the "15" one?
```
help/en/html/doc/Technical/images/GHD_Image_10_PublishBranch.png
help/en/html/doc/Technical/images/GHD_Image_11_NewClass.png
help/en/html/doc/Technical/images/GHD_Image_12_CreateCommit.png
help/en/html/doc/Technical/images/GHD_Image_12_PushCommit.png
help/en/html/doc/Technical/images/GHD_Image_13_CreatePullRequest.png
help/en/html/doc/Technical/images/GHD_Image_15_CreatePullRequest.png
help/en/html/doc/Technical/images/GHD_Image_16_CreatePullRequestDone.png
help/en/html/doc/Technical/images/GHD_Image_17_Conflict.png
help/en/html/doc/Technical/images/GHD_Image_18_Conflict.png
help/en/html/doc/Technical/images/GHD_Image_19_Conflict.png
help/en/html/doc/Technical/images/GHD_Image_1_Clone.png
help/en/html/doc/Technical/images/GHD_Image_20_ConflictSolving.png
help/en/html/doc/Technical/images/GHD_Image_21_ConflictSolving.png
help/en/html/doc/Technical/images/GHD_Image_22_ConflictSolving.png
help/en/html/doc/Technical/images/GHD_Image_2_Clone.png
help/en/html/doc/Technical/images/GHD_Image_3_GitHubDesktop.png
help/en/html/doc/Technical/images/GHD_Image_4_MergeInto.png
help/en/html/doc/Technical/images/GHD_Image_5_MergeInto.png
help/en/html/doc/Technical/images/GHD_Image_6_MergeIntoDone.png
help/en/html/doc/Technical/images/GHD_Image_7_NewBranch.png
help/en/html/doc/Technical/images/GHD_Image_8_NewBranchName.png
help/en/html/doc/Technical/images/GHD_Image_9_NewBranch_Development.png
```

Reply:
The file is in the PR, but has the wrong name.
`help/en/html/doc/Technical/images/Image_14_CreatePullRequest.png`

Reply:
Thanks! Fixed, I think.